components/hab/src/command/studio/docker.rs

```diff
@@ -270,7 +270,6 @@ where
     cmd_args.push(image_identifier(&docker_cmd).into());
     cmd_args.extend_from_slice(args.as_slice());
     if is_serving_windows_containers(&docker_cmd) {
-        cmd_args.push("-w".into());
```

Comment:
Not necessarily relevant to the goal of this PR, but it would be nice for us to use long options whenever possible for readability.